src/main/java/net/dv8tion/jda/internal/managers/EmoteManagerImpl.java

```diff
@@ -112,6 +112,7 @@ public EmoteManagerImpl setName(@Nonnull String name)
         Checks.notBlank(name, "Name");
         name = name.trim();
         Checks.notEmpty(name, "Name");
+        Checks.notShorter(name, 2, "Name");
```

Comment:
Doesn't make much sense to check empty and min length

Reply:
Doesn't make sense for me either, I didn't really look at the existing checks, which was probably a bad idea.